articles/iot-edge/how-to-install-iot-edge-linux.md

```diff
@@ -260,13 +260,15 @@ Run an automated check for the most common configuration and networking errors:
 sudo iotedge check
 ```
 
-And, list running modules:
+>Until you deploy your first module to IoT Edge on your device, the **$edgeHub** system module will not be deployed to the device. As a result, the automated check will return an error for the `Edge Hub can bind to ports on host` connectivty check. This error can be ingored unless it occurs after deploying a module to the device.
```

Comment:
Please remove the `>` at the beginning of this line because it won't render correctly with our markdown engine. 

Typo: "connectivty"

Reply:
Removed and fixed typo. I should have guessed that this wasn't done for good reason 😄